TACC/Core-Portal
# fix:(ui-patterns): show actual class name passed

## Overview

In UI Patterns, for Sections, render user actual class names passed, not the compiled classnames used by the app.

## Related

None

## Changes

- do not pass compiled classname
- pass actual classname

## Testing

1. Open UI Patterns.
2. Find Section patterns.
3. Confirm the following description lists do not show `_some-class_9asdf8`, but just `some_class`:
    - Scrollable
    - With a Complex Table
    - All Properties

## UI

Not convenient to provide. Environment broken. I noticed the bug while working on tup-ui's ui-patterns, fixed it there, then ported the fix to here.
